src/clr-addons/slider/slider.html

```diff
@@ -0,0 +1,56 @@
+<!--
+  ~ Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+<div class="slider">
+  <h4>Value: {{_value}}</h4>
+  <div class="slider-action-field input" [hidden]="!_showsLabels">
+    <input class="value-boundary"
+           title="Minimum"
+           type="number"
+           name="Minimum"
+           [(ngModel)]="_minValue"
+           [readonly]="![enableValueFields]"
+           (change)="onMinValueChange($event.target.value)">
+  </div>
+
+  <!-- TODO:-->
+  <!-- .) orient does not seem to get bound to the sliders' base class orientation-->
+  <div class="slider-action-field">
+    <div
+      [class.slider-wrapper]="_orientation == 'vertical'"
+      [class.slider-size-sm]="_size == 'sm'"
+      [class.slider-size-md]="_size == 'md'"
+      [class.slider-size-lg]="_size == 'lg'"
+      [class.slider-size-xl]="_size == 'xl'">
+      <input #slider
+             class="slider-content"
+             [class.slider-vertical]="_orientation == 'vertical'"
+             [class.slider-horizontal]="_orientation == 'horizontal'"
+             [class.slider-size-sm]="_size == 'sm'"
+             [class.slider-size-md]="_size == 'md'"
+             [class.slider-size-lg]="_size == 'lg'"
+             [class.slider-size-xl]="_size == 'xl'"
+             [(ngModel)]="_value"
+             (change)="valueChanged($event.target.value)"
+             [(step)]="_step"
+             type="range"
+             [(min)]="_minValue"
+             [(max)]="_maxValue"
+             class="slider"
+             [value]="_value">
```

Comment:
binding to value AND ngModel does not make a lot of sense - either one should be good enough. probably value since you are doing something else on your (change) event